lib/puppet/pops/parser/slurp_support.rb

```diff
@@ -45,9 +45,6 @@ def slurp_dqstring
    # Copy from old lexer - can do much better
    def slurp_uqstring
      scn = @scanner
-     last = scn.matched
-     ignore = true
```

Comment:
@hlindberg @thallgren I assume we're trying trying to call accessor methods, e.g. `self.last = scn.matched`?

Reply:
Removed